cvmfs/catalog_mgr_impl.h

```diff
@@ -1,3 +1,13 @@
+/*
+ * catalog_mgr_impl.h
+ *
+ *  Created on: Aug 13, 2015
+ *      Author: molina
+ */
+
+#ifndef CVMFS_CATALOG_MGR_IMPL_H_
+#define CVMFS_CATALOG_MGR_IMPL_H_
+
 /**
```

Comment:
Hm, I don't know what happened.  But if I try to merge, I'd merge with both comment blocks.